docs/ruleset_functions/get_SHW_equipment_associated_with_each_swh_distribution_system.md

```diff
@@ -0,0 +1,46 @@
+## get_SWH_equipment_associated_with_each_swh_distribution_system
+
+Description: This function gets all the SWH equipment connected to a SWH distribution system.  The information is stored in a dictionary where the keys are the SWH Distribution System Ids and values are a dictionary giving the ServiceWaterHeatingEquipment, and Pumps connected to the particular use type  
+
+Inputs:
+- **RMD**
+
+Returns:
+- **swh_and_equip_dict**: A dictionary containing where the keys are the SWH Distribution System and values are dictionaries where keys are the type of SWH equipment and values are the ids of the connected equipment.  Example:  
+{"SWH_Distribution1":{"SWHHeatingEq":["swh_eq1","swh_eq2"], "Pumps":["p1"], "Tanks":["t1"], "Piping":["piping1"], "SolarThermal":[], "SPACE_USES":["sp1_use","sp2_use"], "SPACES_SERVED":[sp1,sp2]}}
+
+Function Call:
+
+- get_obj_by_id
+- get_all_child_SWH_piping_ids
+
+Data Lookup: None
+
+Logic:
+- create a blank dictionary: `swh_and_equip_dict = {}`
+- for each swh distribution system in the RMD: `for distribution in RMD.service_water_heating_distribution_systems:`
+    - add the blank dictionary for the distribution system id: `swh_and_equip_dict.set_default(distribution.id, {"SWHHeatingEq":[],"Pumps":[],"Tanks":[], "Piping":[], "SolarThermal":[], "SPACE_USES":[]})`
+    - now we need to get all of the equipment connected to this distribution system.
```

Comment:
No blank list for SPACES_SERVED? Maybe not needed. Just commenting in case exclusion was a mistake.